backend/README.md

```diff
@@ -1,40 +1,56 @@
 Tobira Backend
 ==============
 
+GraphQL Schema
+--------------
 
-Building the Tobira Backend
----------------------------
-
-Use [Cargo](https://doc.rust-lang.org/cargo/getting-started/first-steps.html) to download all necessary dependencies and to build the backend.
-
-Build and run the backend:
+Export the GraphQL schema of the API.
+The frontend needs this.
 
 ```sh
-cargo run
+cargo run --bin export-schema -- ../frontend/src/schema.graphql
 ```
 
-You can export the GraphQL schema of the API as a file with this command:
 
-```rust
-cargo run --bin export-schema -- ../frontend/src/schema.graphql
-```
+Building the Backend
+--------------------
 
-This puts the generated schema in the correct folder expected by the frontend.
-See also [here](../frontend/README.md).
+Use [Cargo](https://doc.rust-lang.org/cargo/getting-started/first-steps.html) to build the backend.
+This needs the frontend to be built.
 
+```sh
+# build only
+cargo build
+# build and run
+cargo run
+```
 
 Configuration
 -------------
 
-Create `config.toml` in the directory you are running the backend from.
-The file should look like this:
-
+The backend loads configuration from `config.toml` in the current working directory.
 
 ```toml
```

Comment:
That's not directly related to this PR, but I just noticed it, because it touches it, so here goes: Should we really duplicate the entire config file in the docs? :thinking:

Reply:
It's not duplicated since we don't yet have a configuration file anywhere and it needs to be documented somewhere outside the code.

Reply:
Ooh, I didn't even realize that. x)

WHY DON'T WE HAVE THAT, THOUGH?! :scream: 

But okay, sure, that's out of scope for this PR.

Reply:
I'm fine with moving that somewhere else. I just wasn't sure where to. Let's shortly talk about that at the next developers meeting? Moving it should be easy enough.